exp/internal/controllers/machinepool_controller_phases_test.go

```diff
@@ -46,6 +46,8 @@ func init() {
 	externalReadyWait = 1 * time.Second
 }
 
+var timeout = 15 * time.Second
```

Comment:
nit: I wonder if we should inline the timeout given that it's currently a package global (test) variable without being used elsewhere but with a very generic name

Reply:
I'll put it in the test itself if that works?

Reply:
Sure

Reply:
Just felt a bit global before :)